addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ExtensionParamMiner.java

```diff
@@ -80,20 +99,37 @@ private ParamMinerPanel getParamMinerPanel() {
         return paramMinerPanel;
     }
 
+    public ParamMinerConfig getParamMinerConfig() {
+        if (config == null) {
+            config = new ParamMinerConfig();
+        }
+        return config;
+    }
+
     private ZapMenuItem getMenu() {
         if (menu == null) {
             menu = new ZapMenuItem(PREFIX + ".topmenu.tools.title");
 
             menu.addActionListener(
                     e -> {
-                        View.getSingleton()
-                                .showMessageDialog(
-                                        Constant.messages.getString(PREFIX + ".topmenu.tools.msg"));
+                        showParamMinerDialog(null);
```

Comment:
shouldnt you be passing the SiteNode across here?

Reply:
Uhh this is the direct case where I jump into param miner without a prior request or anything. Kind of like the Spider :)